### PR TITLE
added windows server 2022

### LINKED ISSUE #11029: Server 2022
Is there a reason Server 2022 is not listed? please add it if appropriate

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: 2a149789-67c1-e126-129b-6e77e831086d
* Version Independent ID: dd04d845-c098-46e5-fedf-92ba8fc64615
* Content: [Trusted Platform Module Technology Overview (Windows) - Windows security](https://learn.microsoft.com/en-us/windows/security/information-protection/tpm/trusted-platform-module-overview)
* Content Source: [windows/security/information-protection/tpm/trusted-platform-module-overview.md](https://github.com/MicrosoftDocs/windows-itpro-docs/blob/public/windows/security/information-protection/tpm/trusted-platform-module-overview.md)
* Product: **windows-client**
* Technology: **itpro-security**
* GitHub Login: @Dansimp
* Microsoft Alias: **dansimp**

### PR DESCRIPTION
**Pull request by RAJU2529**
as per user report #11029 so i added **Windows Server 2022** after verifying below  official article

**https://learn.microsoft.com/windows-server/get-started/whats-new-in-windows-server-2022**

fixes #11029

### CODE REVIEW

File: windows/security/information-protection/tpm/trusted-platform-module-overview.md [hunk 1]
```diff
@@ -78,10 +79,10 @@ Some things that you can check on the device are:
...
 ## Supported versions for device health attestation
 
-| TPM version | Windows 11  | Windows 10  | Windows Server 2016 | Windows Server 2019 |
-|-------------|-------------|-------------|---------------------|---------------------|
-| TPM 1.2     |             | >= ver 1607 |    >= ver 1607      |       Yes           |
-| TPM 2.0     |    Yes      |     Yes     |       Yes           |       Yes           |
+| TPM version | Windows 11  | Windows 10  | Windows Server 2022 | Windows Server 2019 | Windows Server 2016 |
+|-------------|-------------|-------------|---------------------|---------------------|---------------------|
+| TPM 1.2     |             | >= ver 1607 |                     |                     |      >= ver 1607    |  
+| TPM **2.0** |   **Yes**   |  **Yes**    |    **Yes**          |    **Yes**          |    **Yes**          |
```
Comment (JohanFreelancer9) [inline windows/security/information-protection/tpm/trusted-platform-module-overview.md:85]:
```suggestion
| TPM 2.0 |   **Yes**   |  **Yes**    |    **Yes**          |    **Yes**          |    **Yes**          |
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@Dansimp Please merge this PR. Thanks

**Review by aczechowski — APPROVED** [review]

**Comment by aczechowski** [comment]
Thanks @RAJU2529 for the edit! I made a minor tweak to add back "yes" for Server 2019 and TPM 1.2, which was previously listed.

**Pull request merged**